mantidimaging/gui/windows/spectrum_viewer/roi_table_model.py

```diff
@@ -111,6 +144,8 @@ def headerData(self, section, orientation, role):
                     return "ROI"
                 if section == 1:
                     return "Colour"
+                if section == 2:
+                    return "Visible"
```

Comment:
Can we leave the column header blank, or `' '` , I think its clear enough with out a label and would save some space